lib/matplotlib/transforms.py

```diff
@@ -109,14 +106,17 @@ def __str__(self):
 
     def __getstate__(self):
         d = self.__dict__.copy()
-        # turn the weakkey dictionary into a normal dictionary
-        d['_parents'] = dict(six.iteritems(self._parents))
+        # turn the dictionary with weak values into a normal dictionary
+        d['_parents'] = dict((k, v()) for (k, v) in
```

Comment:
I currently check on deserializing (`__setstate__`) where you *have* to deal with it.  No real need to do it in both places unless we *really* care about the size of the serialization.